src/FSharp.Core/list.fsi

```diff
@@ -1173,6 +1316,9 @@ module List =
     val iter2: action:('T1 -> 'T2 -> unit) -> list1:'T1 list -> list2:'T2 list -> unit
 
     /// <summary>Applies the given function to each element of the collection. The integer passed to the
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence